.github/workflows/test.yml

```diff
@@ -33,6 +33,14 @@ jobs:
         exclude:
         # https://github.com/ruby/setup-ruby/issues/496
         - { os: ubuntu-22.04, ruby: '2.2' }
+        # Too old, fails to compile on macos-14 (arm64)
```

Comment:
`fails to compile`? Do you have an example error?
Since we're not compiling these Rubies on macos-14 do you mean C extensions fail to compile?

Reply:
Ah it's just the same as `# Too old, fails to compile on macos-arm-oss` below, I'll tweak the comment post-merge